src/app/profile/manageGiving/editRecurringGifts/editRecurringGifts.modal.tpl.html

```diff
@@ -0,0 +1,110 @@
+<div ng-switch="$ctrl.state">
+  <div class="modal-header" ng-if="$ctrl.state === 'loading' || $ctrl.state === 'error'"><!-- TODO: use ng-switch-when-separator="|" in Angular 1.5.9 -->
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-xs-12">
+          <div class="border-bottom-small">
+            <button type="button" class="close" aria-label="Close" ng-click="$ctrl.dismiss()"><span aria-hidden="true">&times;</span></button>
+            <h3>Checking Payment Methods</h3>
+            <p>You must have at least one valid payment method to continue</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body" ng-switch-when="loading">
+    <div class="container-fluid" tabindex="-1" role="dialog">
```

Comment:
Ya, idk why they were there. $uibModal seems to add them to the outer modal div. I've removed them.